tsl/test/sql/cagg_joins.sql.in

```diff
@@ -153,6 +162,20 @@ SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
 FROM conditions JOIN devices USING (device_id)
 GROUP BY name, bucket;
 
+SELECT * FROM conditions_summary_daily_3_reorder ORDER BY bucket;
+
+CREATE MATERIALIZED VIEW conditions_summary_daily_3_reorder_using_realtime
+WITH (timescaledb.continuous, timescaledb.materialized_only = FALSE) AS
+SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
+   AVG(temperature),
+   MAX(temperature),
+   MIN(temperature),
+   name
+FROM conditions JOIN devices USING (device_id)
+GROUP BY name, bucket;
```

Comment:
Would be good to also add a regression test for a cagg with explicit JOIN and a WHERE clause... also failing tests if, for example, you add a explicit JOIN + WHERE but with other failure conditions (join type, number of join conditions, join condition operator)